### PR TITLE
Support for 3044 and several smaller fixes

### PR DESCRIPTION
**Pull request by eliasp**
I added support for [Avery Zweckform 3044 Mini ](https://www.avery-zweckform.com/produkt/vielzweck-etiketten-3044) labels and improved/fixed a few minor things.
See the commits for more details.

### CODE REVIEW

File: AveryLabels.py [hunk 1]
```diff
@@ -24,18 +21,21 @@
 # page margins left/top
```
Comment (gitolicious) [inline AveryLabels.py:21]:
```suggestion
# page margins left/top
# page size w/h or name from reportlab.lib.pagesizes
```

File: AveryLabels.py [hunk 2]
```diff
@@ -24,18 +21,21 @@
...
 
 labelInfo = {
+    # 22x 32mm x 10mm mini labels
+    3044: ( 2, 11, (32, 10), (2,2), (1, 1), (66.5*mm, 120.5*mm)),
+
     # 2.6 x 1 address labels
-    5160: ( 3, 10, (187,  72), (11, 0), (14, 36)),
-    5161: ( 2, 10, (288,  72), (0, 0), (18, 36)),
+    5160: ( 3, 10, (187,  72), (11, 0), (14, 36), LETTER),
```
Comment (gitolicious) [inline AveryLabels.py]: Actually my label sheets are `A4` size, not `LETTER`. Unsure if there are actually letter sized versions in the US, but let's use A4 until someone finds them.

**Review by gitolicious — CHANGES_REQUESTED** [review]
Thanks a lot for this improvement. Just some minor thoughts from my side...

File: AveryLabels.py (hunk 2 above)
Comment (eliasp) [reply AveryLabels.py]: I used `LETTER`, since this was the default (`self.pagesize = LETTER`) until now. Replaced it all with `A4` now.

Comment (gitolicious) [reply AveryLabels.py]: Yeah, I realized that as well after you copied it to the label definitions. I blindly copied from the original at https://gist.github.com/timrprobocom/3946aca8ab75df8267bbf892a427a1b7#file-averylabels-py-L52
Thanks for the change.

**Pull request merged**